debezium-server-eventhubs/src/main/java/io/debezium/server/eventhubs/EventHubsChangeConsumer.java

```diff
@@ -86,12 +90,14 @@ void connect() {
         maxBatchSize = config.getOptionalValue(PROP_MAX_BATCH_SIZE, Integer.class).orElse(0);
         configuredPartitionId = config.getOptionalValue(PROP_PARTITION_ID, String.class).orElse("");
         configuredPartitionKey = config.getOptionalValue(PROP_PARTITION_KEY, String.class).orElse("");
+        hashMessageKey = config.getOptionalValue(PROP_HASH_MESSAGE_KEY, Boolean.class).orElse(false);
```

Comment:
There is no need for this config option. Either hash key function is set and hashing is then done or not.